packages/components/src/DateTimePickers/pickers/IncrementableScrollActionList/IncrementableScrollActionList.component.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IncrementableScrollList from './IncrementableScrollList';
+import PickerAction from '../../PickerAction';
+
+class IncrementableScrollActionList extends React.Component {
+
+	constructor(props) {
+		super(props);
+
+		const middleInitialIndex = this.props.items.findIndex(
+			item => item.id === this.props.initialMiddleVisibleItemId);
+
+		this.initialIndex = middleInitialIndex === -1
+			? 0
+			: Math.max(middleInitialIndex - 2, 0);
```

Comment:
You can remove the ternary and write `this.initialIndex = Math.max(middleInitialIndex - 2, 0)`

Reply:
Yeah we could, but i find it less obvious that the 'not find item' case is taken care. It sort of mix 2 things